docs/deployment.md

```diff
@@ -0,0 +1,90 @@
+# Deployments
+
+## Next.js app
+
+The Next.js app can be deployed as a static HTML export or as a Node.js server, depending on your needs. The [Next.js deployment documentation](https://nextjs.org/docs/deployment) provides more information on how to deploy your application.
+
+## Storybook
+
+Storybook can be deployed as a static HTML export. The [Storybook deployment documentation](https://storybook.js.org/docs/react/sharing/publish-storybook) provides more information on deployment options.
+
+### GitHub Pages
+
+[GitHub Pages](https://pages.github.com/) may be a good option for hosting Storybook if you only need one environment to represent the latest state of your main branch.
+
+**Steps to setup GitHub Pages**
+
+1. Add a workflow to automatically deploy Storybook: `.github/workflows/deploy-storybook.yml` (see example below)
+1. Enable the "GitHub Actions" source in your rep settings (`Settings > Pages > Source`)
+1. Trigger the workflow from the Actions tab or by pushing a commit to your main branch
+
+---
+
+<details>
+  <summary>View GitHub workflow</summary>
+
+```yaml
+name: Deploy Storybook
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow access to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Cancel any older in-progress runs of this workflow
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+          cache: npm
+          cache-dependency-path: ./app/package-lock.json
```

Comment:
jfyi: I ham-fistedly smushed this into my repo which is using `yarn` instead of `npm` and trying to convert this to `cache: yarn` failed for me. Not sure if it's helpful to know, but I thought I'd share.

Reply:
@rocketnova Added a code comment here about what you'd want if using Yarn: https://github.com/navapbc/template-application-nextjs/pull/89/commits/24410b5016ebe4ab89c2a010b1c9049daf6fda22

Reply:
Thank you!